### PR TITLE
Attempt to make unique bank card designs

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by maxwofford** [comment]
Screen Shot 2021-04-10 at 10 08 44

Seems like the preview worked

**Comment by motdotla** [comment]
Thanks team. Going to table this for now though. Trying to simplify/unify UX - so moving to a table based layout on this that will be searchable soon. Shipping status will also move into this table to simplify that UX.

**Comment by sampoder** [comment]
Yep it worked but the CSS is acting weird?

```
background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.75)), url("https://bank--pr-1451.herokuapp.com/my/cards");
```

I don't know how exactly Rails works but its making it this which is weird

Separate question: is there any way to get a non sensitive unique id for a card? like the number in the URL

**Comment by garyhtou** [comment]
Yeah, every card has an `id` from stripe that looks something like this: `"card_1IerSyFSaumjmb9rwCVGUdyY"`
```erb
<%= stripe_card.stripe_id %>
```

or, to get the `id` in the URL, I think you can just do
```erb
<%= stripe_card.id %>
```

**Comment by sampoder** [comment]
Ahh great! Thanks Gary!

File: app/views/stripe_cards/_stripe_card.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <a class="text-decoration-none" href="<%= stripe_card_path(stripe_card) %>">
 
   <%= creator_bar stripe_card, prefix: 'issued ' unless defined? headless %>
-  <div class="stripe-card <%= stripe_card.status_text %> <%= stripe_card.virtual? ? 'virtual' : '' %>" data-behavior="hover_tilt">
+  <div class="stripe-card <%= stripe_card.deactivated? ? 'deactivated' : '' %> <%= stripe_card.virtual? ? 'virtual' : '' %>" data-behavior="hover_tilt" style="background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.75)), url("https://workshops.hackclub.com/api/patterns/<%= stripe_card %>)">
```
Comment (sampoder) [inline app/views/stripe_cards/_stripe_card.html.erb]:
```suggestion
  <div class="stripe-card <%= stripe_card.deactivated? ? 'deactivated' : '' %> <%= stripe_card.virtual? ? 'virtual' : '' %>" data-behavior="hover_tilt" style="background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.75)), url("https://workshops.hackclub.com/api/patterns/<%= stripe_card.id %>)">
```

**Pull request closed without merging**